angular-workspace/projects/ni/nimble-angular/table-column/number-text/tests/nimble-table-column-number-text.directive.spec.ts

```diff
@@ -77,6 +77,7 @@ describe('NimbleTableColumnNumberText', () => {
                         grouping-disabled
                         format="${NumberTextFormat.decimal}"
                         decimal-digits="6"
+                        alignment=${NumberTextAlignment.left}
```

Comment:
nit: Apparently the quotes are not necessary, but just for consistency with above
```suggestion
                        alignment="${NumberTextAlignment.left}"
```